src/null-safety/index.md

```diff
@@ -334,6 +334,18 @@ If you find any issues with null safety please [give us feedback.][]
 
 [language version]: /guides/language/evolution#language-versioning
 
+## Known issues
+
+Not all parts of the Dart SDK support null safety yet,
+as some parts still need additional work to
```

Comment:
Maybe this (and then we can modify "issues" below to make clear that we're only going to talk about the Dart team's stuff):

```suggestion
Some parts of the Dart ecosystem still need additional work to
```